### PR TITLE
Fix/rewards added feedback

### PR DESCRIPTION
**Pull request by divyam1705**
- Feedback added upon reward addition.

### CODE REVIEW

File: packages/frontend/src/assets/bordered-tick.icon.tsx
Comment (guerrap) [inline packages/frontend/src/assets/bordered-tick.icon.tsx:1]: Let's name the file bordered-tick-icon.tsx`.

File: packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx [hunk 1]
```diff
@@ -179,6 +181,11 @@ export function RewardsStep({
             : [reward];
 
         setOpen(false);
+        setFeedbackVisible(true);
+        setTimeout(() => {
```
Comment (guerrap) [inline packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx:185]: Wondering if we should move the timeout to a `useEffect` so we can also clear it on unmount, for example:

```tsx
useEffect(() => {
    if (!feedback) return;

    const timeout = setTimeout(() => {
        setFeedbackVisible(false);
    }, 1000);

    return () => clearTimeout(timeout);
}, [feedback]);
```
Wdyt?

File: packages/frontend/src/assets/bordered-tick.icon.tsx
Comment (divyam1705) [reply packages/frontend/src/assets/bordered-tick.icon.tsx:1]: oops typo

File: packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx (hunk 1 above)
Comment (divyam1705) [reply packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx:185]: Yes, I'll do it.

**Review by guerrap — APPROVED** [review]

File: packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx (hunk 1 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx]: Let's have `1500` for the feedback here, to be consistent with the address copy.

**Review by guerrap — APPROVED** [review]

**Pull request merged**